pkg/executor/backoff_test.go

```diff
@@ -37,7 +37,7 @@ var _ = Describe("exponential limited backoff", func() {
 
 	BeforeEach(func() {
 		testsClock = clock.NewFakeClock(time.Time{})
-		backoff = executor.NewExponentialLimitedBackoffWithClock(executor.DefaultMaxStep, testsClock)
+		backoff = executor.NewExponentialLimitedBackoffWithClockWithOptions(testsClock)
```

Comment:
Hi @RamLavi, is it possible to please add unit tests for the new framework functionality that you've added?

Reply:
sure, refactored the unit test section to test both no-opts and custom-opts. [Done](https://github.com/kubevirt/kubevirt/compare/6108194647fafdfbbf659cafc62f1cc7e04f927a..39d8d088e434a5045fe2ad8f9edc3a00aaed2058)